components/xwiki-platform-users/xwiki-platform-users-wiki/src/main/java/org/xwiki/users/internal/WikiUser.java

```diff
@@ -94,10 +94,10 @@ public String getName()
         if (this.profileReference == null) {
             return "";
         }
-        String result = this.bridge.getProperty(this.profileReference, this.classReference, "first_name") + "";
+        String result = this.bridge.getProperty(this.profileReference, this.classReference, "first_name");
```

Comment:
You're right. I forgot to compile the project again after this last change. I will force-push a fix momentarily.